libtransmission/torrent-magnet.cc

```diff
@@ -304,7 +304,7 @@ static void onHaveAllMetainfo(tr_torrent* tor, tr_incomplete_metadata* m)
         tor->incompleteMetadata = nullptr;
         tor->isStopping = true;
         tor->magnetVerify = true;
-        if (tor->session->shouldPauseAddedTorrents())
+        if (tor->session->shouldPauseAddedTorrents() && !tor->startAfterVerify)
```

Comment:
I don't think this can be right? This change says "only set startAfterVerify to false if it was already false"

Reply:
FWIW i tested all states on macOS and this does work

Reply:
What is the default value of startafterverify for a new torrent? I didnt follow it all the way to creation, I assumed it was not set… but if it is actually false…

ill check when i get home later today

Reply:
If's false by default; initialized where it's declared

```c++
torrent.h
891:    bool startAfterVerify = false;
```

Reply:
> Does it not mean “set to false” if it does not exist?

This gets me everytime 🤦‍♂️

> If's false by default; initialized where it's declared
> 
> ```c++
> torrent.h
> 891:    bool startAfterVerify = false;
> ```

Ill fix up the PR in a bit. Currently on a food mission